complex-events/gather_stats.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+from collections import Counter
+import os
+
+import hgvs.parser
+
+from consequence_prediction.repeat_expansion_variants.clinvar_identifier_parsing import parse_variant_identifier
+from eva_cttv_pipeline.clinvar_xml_utils import ClinVarDataset, find_elements
+
+hgvs_parser = hgvs.parser.Parser()
+
+
+def counts(input_xml, output_dir):
+    """
+    Gather counts for variant descriptors present in input_xml.
+    Returns counts and outputs a text file of all unparseable hgvs descriptors to output_dir.
+    """
+    dataset = ClinVarDataset(input_xml)
+
+    variant_type_hist = Counter()
+    total_count = 0
+    spdi_count = 0
+    hgvs_count = 0
+    toplevel_refseq_hgvs_count = 0
+    cytogenetic_count = 0
+    start_stop_count = 0
+    strict_parseable_hgvs_count = 0
+    our_parseable_hgvs_count = 0
+    any_parseable_hgvs_count = 0
+    hgvs_only_count = 0
+    cytogenetic_only_count = 0
+    start_stop_only_count = 0
+
+    hgvs_output_file = open(os.path.join(output_dir, 'unparseable-hgvs.txt'), 'w+')
+
+    for record in dataset:
+        if record.measure:
```

Comment:
I think it would be better to invert this if condition and use a continue, like you do for m.has_complete_coordinates, to reduce nesting in this loop.